src/elements/header-toolbar.html

```diff
@@ -347,6 +370,19 @@
           '--hero-logo-color': settings.backgroundImage ? '#fff' : 'var(--default-primary-color)',
         });
       }
+
+      _onAddToHomescreen() {
+        this.addToHomescreen.prompt();
+        this.addToHomescreen.userChoice
+          .then((choiceResult) => {
+            if (choiceResult.outcome === 'accepted') {
+              ga('send', 'event', 'prompt', 'accepted');
```

Comment:
shall we specify in details what kind of prompt? e.g. "add to home screen prompt"

Reply:
we have only one prompt right now, but sure we can be more specific